.env

```diff
@@ -1,4 +1,19 @@
-NODE_ENV=dev
+# Images (generally you want to use the same tag for all @ceramicnetwork images)
+CERAMIC_IMAGE_TAG=latest
+CAS_IMAGE_TAG=latest
+IPFS_IMAGE_TAG=latest
+
+# Ceramic
+CERAMIC_DAEMON_PORT=7007
+CERAMIC_DEBUG=true
+CERAMIC_NETWORK=local
+CERAMIC_ETH_RPC=http://ganache:8545
+
+# Ceramic
+CERAMIC_DAEMON_PORT=7007
+CERAMIC_DEBUG=true
+CERAMIC_NETWORK=local # Or try "testnet-clay"
+CERAMIC_ETH_RPC_URL=http://ganache:8545
```

Comment:
Is this some hostname internal to docker?

Reply:
yep